trade-finance-manager-api/api-tests/v1/deals/deals-submit.api-test.js

```diff
@@ -41,6 +41,15 @@ const getGefMandatoryCriteriaByVersion = jest.fn(() => Promise.resolve([]));
 api.getGefMandatoryCriteriaByVersion = getGefMandatoryCriteriaByVersion;
 
 describe('/v1/deals', () => {
+  beforeAll(() => {
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date('2024-06-28'));
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
```

Comment:
This change should hopefully avoid the flakey result where the time-related field in the response can sometimes be off by around 1 ms